### PR TITLE
Add a new "Use" step to lifecycle.

### PR DESCRIPTION
**Pull request by paulboosz**
[User Story](https://www.notion.so/Afficher-l-impact-d-utilisation-dans-la-webapp-b98cafcdc799428b8ecdbe0313b88148)

### CODE REVIEW

**Comment by n1k0** [comment]
I think the patch is a state where it could start being reviewed. Lot of changes come from the JSON files, so don't be too afraid with the diff stats @magopian 😄

**Comment by paulboosz** [comment]
for my last commit cc16b63, I had to change the tool I use to convert Gsheet -> Json. This results in some confusing changes in numbers formatting.

File: src/Server/Query.elm
Comment (magopian) [inline src/Server/Query.elm:215]: Is 100 a random number, or one that we get from the "base impact"? Should we document that somewhere (also maybe in the [openapi.yaml](https://github.com/MTES-MCT/wikicarbone/pull/61#diff-d910ba2ef878f7db0223a966b81c8b3f3b65027bb39e4431bb05140171eece39R256)?)

File: openapi.yaml
Comment (magopian) [inline openapi.yaml:256]: Should it also have a `max`, as we validate the max number of cycles in `Server/Query.elm`?

File: tests/Data/SimulatorTest.elm
Comment (magopian) [inline tests/Data/SimulatorTest.elm:45]: Do you want to fix that before merging?

**Review by magopian — APPROVED** [review]
Impressive work! A few nits that you may want to fix before merging ;)

I'll approve anyway as I didn't see anything blocking, and you might want to merge early, and fix later. Keep in mind that I'd like to rebase my PR #57 on this one (or on master once this one is merged) asap ;)

File: src/Server/Query.elm
Comment (n1k0) [reply src/Server/Query.elm:215]: I never had any upper limit specified, so I went with `100` because the max number of cycles defined for products is `45` and I wanted to offer a bit more latitude to inspect variability. I'll add documentation in `openapi.yml` about that.

Comment (n1k0) [reply src/Server/Query.elm:215]: Actually the lower & upper limits were already defined in openapi docs.

File: openapi.yaml
Comment (n1k0) [reply openapi.yaml:256]: This is the `customCountryMixMakingParam` param and there's no upper limit defined by specs.

File: tests/Data/SimulatorTest.elm
Comment (n1k0) [reply tests/Data/SimulatorTest.elm:45]: Yes, I'm just waiting for @paulboosz confirmation that computations are correct. Then i'll update all tests with new results and uncomment this.

File: src/Data/Process.elm
Comment (n1k0) [reply src/Data/Process.elm]: Fixed by https://github.com/MTES-MCT/wikicarbone/pull/61/commits/ba45bff231c9ce4d90c71a8db30eac20eacfd33e

File: tests/Data/SimulatorTest.elm
Comment (n1k0) [reply tests/Data/SimulatorTest.elm]: @Got validation from @paulboosz over Mattermost that we can land this. I've uncommented and updated simulation score tests in https://github.com/MTES-MCT/wikicarbone/pull/61/commits/7dd4a3ec350af2abf29967afd78c743cb972fc68.

**Pull request merged**
